frontend/src/board/contextmenu/colors.tsx

```diff
@@ -32,3 +45,10 @@ export function colorsSubMenu({ board, focusedItems, dispatch }: SubmenuProps) {
         dispatch({ action: "item.update", boardId: b.id, items: updated })
     }
 }
+
+function itemColorOrDefault(items: Item[]) {
+    if (items.length === 0) return YELLOW
+    const firstNote = items.find((i): i is Note => i.type === "note")
```

Comment:
Use isColoredItem instead, to support all colorable items

Reply:
Changed!